07-DialogService/src/PrismSample/PrismSample.csproj

```diff
@@ -7,6 +7,10 @@
 
   <ItemGroup>
     <PackageReference Include="Prism.DryIoc.Forms" Version="7.2.0.1422" />
-    <PackageReference Include="Xamarin.Forms" Version="4.3.0.991211" />
+    <PackageReference Include="Xamarin.Forms" Version="4.4.0.991265" />
```

Comment:
no updates please for any existing references.

Reply:
Done